lms/static/js/ccx/schedule.js

```diff
@@ -175,14 +175,14 @@ var edx = edx || {};
             });
       // schedule_template defined globally in ccx\schedule.html
       /* globals schedule_template */
-            this.$el.html(schedule_template({chapters: this.showing}));
+            this.$el.html(schedule_template({chapters: this.showing})); // xss-lint: disable=javascript-jquery-html
```

Comment:
Just for the context, this could've been fixed by using `edx.HtmlUtils.setHtml` but I didn't since "schedule_template defined globally in ccx\schedule.html" and changing that file for the sake of fix might have some unexpected results since the usage for `schedule_template` is different in some files. 

I checked the underscore file and it is safe. https://github.com/edx/edx-platform/blob/master/lms/templates/ccx/schedule.underscore